buildenv/docker/aarch64-linux_CC/Dockerfile

```diff
@@ -53,27 +56,31 @@ RUN apt-get update \
     zip \
   && rm -rf /var/lib/apt/lists/*
 
+# Add user home
+RUN useradd -ms /bin/bash --uid 1000 $user
+RUN echo ulimit -u 32000 -n 2048 >> /home/$user/.bashrc
```

Comment:
Why do we want to reduce these limits? For `-u` it would otherwise be unlimited, for `-n` the limit would be 1048576.

Reply:
I copied it from what mkdocker.sh is doing, I can just remove it.

Reply:
That was added in #11439. I think we should remove it (here and in mkdocker.sh) unless there's a compelling reason for it to stay.

Reply:
Updated.